src/app/test/db/mongo/models_test.py

```diff
@@ -28,6 +30,8 @@
     UserToken,
 )
 
+enable_gridfs_integration()
```

Comment:
Something worth noting here: there's a fundamental incompatibility when using this `enable_gridfs_integration` that is going to require us to reorganize the unit tests to get around.  With this in here, it is currently the case that:

* Running the full suite of unit tests works fine
* Running just this test file in isolation does not work
* Running tests for any class in this file in isolation **other than** TestGarden works fine.

The conclusion I came to when trying to resolve this is that `enable_gridfs_integration` establishes some mocks that could break things depending on the order in which things get imported.

This test file is too big and should be broken up anyway, so I think we split out the tests for each model into their own test file and that would solve most of this.  I'm not going to do that here though because I believe there's another branch in progress that started to do this.